CNV#43342: RN - VMs require 1GiB memory for mem hotplug (#78025)
openshift/openshift-docs: virt/release_notes/virt-4-16-release-notes.adoc

+
On {op-system-base-full} 9, xref:../../virt/monitoring/virt-exposing-downward-metrics.adoc#virt-viewing-downward-metrics-cli_virt-exposing-downward-metrics[use the command line] to view downward metrics. The `vm-dump-metrics` tool is not supported on the {op-system-base-full} 9 platform.

[id="virt-4-16-notable-technical-changes"]
=== Notable Technical Changes

//CNV-43342: RN: VMs require 1GiB memory

> * VMs require a minimum of 1 GiB of allocated memory to enable memory hotplug. If a VM has less than 1 GiB of allocated memory, then memory hotplug will be disabled.

Reviewer: ```suggestion
* VMs require a minimum of 1 GiB of allocated memory to enable memory hotplug. If a VM has less than 1 GiB of allocated memory, then memory hotplug is disabled.
```
present-tense only
